### PR TITLE
Corrdiff generation bug fixes & Update README

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

Updating README, changing some configurations, and fixing small bugs

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by jialusui1102 — COMMENTED** [review]
Changes look great!

File: examples/generative/corrdiff/README.md [hunk 1]
```diff
@@ -29,29 +29,87 @@ according to the comments. Alternatively, you can create a new config file
...
+  - model/corrdiff_regression
+
+  # Training
+  - training/corrdiff_regression
+
+  # Validation
+  - validation/basic
+  ```
+
+Similarily, for taining of the diffusion model, you should have:
```
Comment (ktangsali) [inline examples/generative/corrdiff/README.md:88]: Similarily -> Similarly

**Review by ktangsali — APPROVED** [review]

File: examples/generative/corrdiff/README.md (hunk 1 above)
Comment (mnabian) [reply examples/generative/corrdiff/README.md]: Fixed it

**Pull request merged**